CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
We should agree on one consistent WIP label. I am fine with `## [Unreleased]`, but have used `## Dev` for now.

Reply:
Thanks for the heads-up! I didn't know and went with how https://keepachangelog.com/en/1.1.0/ has it in its example. Now I know! :)